environment.rb

```diff
@@ -0,0 +1,7 @@
+require 'bundler/setup'
+require 'dotenv'
+
+Bundler.require :default, ENV['RACK_ENV'].to_sym
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.